interface/wx/uilocale.h

```diff
@@ -80,6 +99,49 @@ class wxUILocale
      */
     static const wxUILocale& GetCurrent();
 
+    /**
+        Create the locale corresponding to the given locale identifier.
```

Comment:
Supernitpick: We had a discussion about verb forms some time ago and it was agreed to use Creates/Sets/Returns... instead of Create/Set/Return..., see the first bullet [here](https://github.com/wxWidgets/wxWidgets/blob/master/docs/contributing/how-to-add-class-documentation.md#important-dos-and-donts).

Reply:
I completely forgot about it, thanks.